wagtail/test/testapp/migrations/00053_alter_formfield_clean_name.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tests", "0052_alter_extendedformfield_clean_name_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='formfield',
```

Comment:
We should be able to merge this test app migration with the previous one. You can do this by deleting both locally and re-running the test app make migrations command. 

https://docs.wagtail.org/en/latest/contributing/developing.html#running-migrations-for-the-test-app-models

Then commit the changes, which should end up with one migration file for the tests.